src/antsibull_nox/sessions/ansible_lint.py

```diff
@@ -13,21 +13,21 @@
 import nox
 
 from .collections import prepare_collections
-from .utils import PackageType, install
+from .utils import PackageType, PackageTypeOrList, install, normalize_package_type
 
 
 def add_ansible_lint(
     *,
     make_ansible_lint_default: bool = True,
-    ansible_lint_package: PackageType = "ansible-lint",
+    ansible_lint_package: PackageTypeOrList = "ansible-lint",
     strict: bool = False,
 ) -> None:
     """
     Add a session that runs ansible-lint.
     """
 
     def compose_dependencies() -> list[PackageType]:
-        return [ansible_lint_package]
+        return list(normalize_package_type(ansible_lint_package))
```

Comment:
Doesn't `normalize_package_type` already return a list?

```suggestion
        return normalize_package_type(ansible_lint_package)
```

Reply:
Yes, but it doesn't own it, and I want to avoid the problem that someone adjusts the code to
```py
        deps = normalize_package_type(ansible_lint_package)
        deps.extend(...)
        return deps
```
Maybe it's better to change the return value to `Sequence[...]` instead of `list[...]`... WDYT?

Reply:
I decided to make `normalize_package_type` return an owned list, and make it accept `Sequence[...]` instead of `list[...]` for more flexibility.